pennylane/ops/qubit.py

```diff
@@ -1661,6 +1661,153 @@ def adjoint(self, do_queue=False):
         return U3(*self.parameters, wires=self.wires, do_queue=do_queue).inv()
 
 
+# =============================================================================
+# Quantum chemistry
+# =============================================================================
+
+
+class SingleExcitation(Operation):
+    r"""SingleExcitation(phi, wires)
+    Single excitation rotation.
+
+    .. math:: U(\phi) = \begin{bmatrix}
+                1 & 0 & 0 & 0 \\
+                0 & \cos(\phi/2) & -\sin(\phi/2) & 0 \\
+                0 & \sin(\phi/2) & \cos(\phi/2) & 0 \\
+                0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+    This operation performs a rotation in the two-dimensional subspace :math:`\{|01\rangle,
+    |10\rangle\}`. The name originates from a fermionic interpretation, where the transformation
+    from :math:`|10\rangle` to `:math:`|01\rangle` is interpreted as "exciting" a particle from
+    the first qubit to the second.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 1
+    * Gradient recipe: Obtained from its decomposition in terms of the
+        :class:`~.SingleExcitationPlus` and `:class:`~.SingleExcitationMinus` operations
```

Comment:
Worth checking that the new indentation is correct (`Gradient recipe` appeared with bold before). If okay, removing the spaces is applicable for two other places too in the PR (where `Gradient recipe` is being defined).
```suggestion
      :class:`~.SingleExcitationPlus` and :class:`~.SingleExcitationMinus` operations
```